static/app/views/performance/traceDetails/transactionGroup.tsx

```diff
@@ -77,34 +79,63 @@ class TransactionGroup extends Component<Props, State> {
       numOfOrphanErrors,
       onlyOrphanErrors,
       isOrphanError,
+      traceViewRef,
     } = this.props;
     const {isExpanded} = this.state;
 
     return (
       <Fragment>
-        <TransactionBar
-          location={location}
-          organization={organization}
-          measurements={measurements}
-          generateBounds={generateBounds}
-          index={index}
-          transaction={transaction}
-          traceInfo={traceInfo}
-          continuingDepths={continuingDepths}
-          isOrphan={isOrphan}
-          isLast={isLast}
-          isExpanded={isExpanded}
-          toggleExpandedState={this.toggleExpandedState}
-          isVisible={isVisible}
-          hasGuideAnchor={hasGuideAnchor}
-          barColor={barColor}
-          addContentSpanBarRef={addContentSpanBarRef}
-          removeContentSpanBarRef={removeContentSpanBarRef}
-          onWheel={onWheel}
-          onlyOrphanErrors={onlyOrphanErrors}
-          numOfOrphanErrors={numOfOrphanErrors}
-          isOrphanError={isOrphanError}
-        />
+        {organization.features.includes('performance-trace-details') ? (
+          <TransactionBarV0
```

Comment:
Might be tedious but I feel like we should pull these props out into an object that we spread into either component, that way we can tell if they have any newly added props, or if they're both the same

Reply:
Yep, did this!